src/card/Card.js

```diff
@@ -78,29 +78,28 @@ const Card = props => {
               style={[{ width: null, height: 150 }, imageStyle && imageStyle]}
               source={image}
             >
-              <View
-                style={[styles.overlayContainer, overlayStyle && overlayStyle]}
-              >
-                {featuredTitle !== null &&
-                  <Text
-                    style={[
-                      styles.featuredTitle,
-                      featuredTitleStyle && featuredTitleStyle,
-                    ]}
-                  >
-                    {featuredTitle}
-                  </Text>}
-                {featuredSubtitle !== null &&
-                  <Text
-                    style={[
-                      styles.featuredSubtitle,
-                      featuredSubtitleStyle && featuredSubtitleStyle,
-                    ]}
-                  >
-                    {featuredSubtitle}
-                  </Text>}
-              </View>
-            </BackgroundImage>
+              {(featuredTitle || featuredSubtitle) &&
+                <View style={styles.overlayContainer}>
+                  {featuredTitle &&
+                    <Text
+                      style={[
+                        styles.featuredTitle,
+                        featuredTitleStyle && featuredTitleStyle,
+                      ]}
+                    >
+                      {featuredTitle}
+                    </Text>}
+                  {featuredSubtitle &&
+                    <Text
+                      style={[
+                        styles.featuredSubtitle,
+                        featuredSubtitleStyle && featuredSubtitleStyle,
+                      ]}
+                    >
+                      {featuredSubtitle}
+                    </Text>}
+                </View>}
+            </Image>
```

Comment:
Supposed to be BackgroundImage

Reply:
Oops the merge went wrong
![giphy](https://user-images.githubusercontent.com/17592779/31140509-e61490d2-a874-11e7-8139-94063ec5d75d.gif)